scripts/docker-build.sh

```diff
@@ -40,4 +40,4 @@ docker rm -f ${BUILD_CONTAINER_NAME}
 
 echo
 echo "Built image ${IMAGE_NAME}:${TAG_NAME}"
-echo "Run 'docker run -d --name some-timescaledb -p 5432:5432 ${IMAGE_NAME}:${TAG_NAME}' to launch"
+echo "Run 'docker run --env TIMESCALEDB_TELEMETRY=off -d --name some-timescaledb -p 5432:5432 ${IMAGE_NAME}:${TAG_NAME}' to launch"
```

Comment:
Again, I think it is much better and safer to set telemetry to off by default in the config file to not burden developers with this env variable. This also avoids having this env variable trickle down into other scripts that depend on the built image.